parse/ConditionPythonParser.cpp

```diff
@@ -22,3 +29,135 @@ return condition_wrapper(std::make_shared<Condition::Not>(
     ));
 }
 
+
+namespace {
+    condition_wrapper insert_owned_by_(const boost::python::tuple& args, const boost::python::dict& kw) {
+        std::unique_ptr<ValueRef::ValueRef<int>> empire;
+        EmpireAffiliationType affiliation = EmpireAffiliationType::AFFIL_SELF;
+
+        if (kw.has_key("empire")) {
+            auto empire_args = boost::python::extract<value_ref_wrapper<int>>(kw["empire"]);
+            if (empire_args.check()) {
+                empire = ValueRef::CloneUnique(empire_args().value_ref);
+            } else {
+                empire = std::make_unique<ValueRef::Constant<int>>(boost::python::extract<int>(kw["empire"])());
+            }
+        }
+
+        if (kw.has_key("affiliation")) {
+            affiliation = boost::python::extract<enum_wrapper<EmpireAffiliationType>>(kw["affiliation"])().value;
+        }
+
+        return condition_wrapper(std::make_shared<Condition::EmpireAffiliation>(std::move(empire), affiliation));
+    }
+
+    condition_wrapper insert_contained_by_(const condition_wrapper& cond) {
+        return condition_wrapper(std::make_shared<Condition::ContainedBy>(ValueRef::CloneUnique(cond.condition)));
+    }
+
+    condition_wrapper insert_contains_(const condition_wrapper& cond) {
+        return condition_wrapper(std::make_shared<Condition::Contains>(ValueRef::CloneUnique(cond.condition)));
+    }
+
+    template <MeterType M>
+    condition_wrapper insert_meter_value_(const boost::python::tuple& args, const boost::python::dict& kw) {
```

Comment:
why `MeterType M` a template parameter, rather than a standard function parameter?

Reply:
It was simple to write. I've changed it now.